aos.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'aos' {

```

Comment:
Why have you added this file? Please explain.

Reply:
@Doraemon012  To configure it for aos library

Reply:
Aos is used for adding aniamtion to it